### PR TITLE
CreateJS: addEventListener can receive a Function

### PR DESCRIPTION
**Pull request by endel**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If changing an existing definition:
- [x] Provide a URL to  documentation or source code which provides context for the suggested changes:         http://createjs.com/docs/easeljs/classes/EventDispatcher.html#method_addEventListener
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: createjs-lib/index.d.ts [hunk 1]
```diff
@@ -70,6 +70,7 @@ declare namespace createjs {
         addEventListener(type: string, listener: (eventObj: Object) => void, useCapture?: boolean): Function;
         addEventListener(type: string, listener: { handleEvent: (eventObj: Object) => boolean; }, useCapture?: boolean): Object;
         addEventListener(type: string, listener: { handleEvent: (eventObj: Object) => void; }, useCapture?: boolean): Object;
+        addEventListener(type: string, listener: Function, useCapture?: boolean): Function;
```
Comment (andy-ms) [inline createjs-lib/index.d.ts]: Is there a more specific type than `Function` you can use?

**Comment by andy-ms** [comment]
@endel Rebase onto the latest `types-2.0` to fix tests.

File: createjs-lib/index.d.ts (hunk 1 above)
Comment (sandersn) [inline createjs-lib/index.d.ts]: `(...args: any[]) => any` is stricter than `Function`, for example, since it disallows `new Function("...")`

**Comment by paulvanbrenk** [comment]
This PR seems abandoned, please re-open when you made the requested changes.

**Pull request closed without merging**